dataplane/linux/bpf_ep_mgr.go

```diff
@@ -967,3 +881,87 @@ func (m *bpfEndpointManager) isWorkloadIface(iface string) bool {
 func (m *bpfEndpointManager) isDataIface(iface string) bool {
 	return m.dataIfaceRegex.MatchString(iface)
 }
+
+func (m *bpfEndpointManager) addWEPToIndexes(wlID proto.WorkloadEndpointID, wl *proto.WorkloadEndpoint) {
+	for _, t := range wl.Tiers {
+		m.addPolicyToWEPMappings(t.IngressPolicies, wlID)
+		m.addPolicyToWEPMappings(t.EgressPolicies, wlID)
+	}
+	m.addProfileToWEPMappings(wl.ProfileIds, wlID)
+}
+
+func (m *bpfEndpointManager) addPolicyToWEPMappings(polNames []string, wlID proto.WorkloadEndpointID) {
+	for _, pol := range polNames {
+		polID := proto.PolicyID{
+			Tier: "default",
```

Comment:
I believe we've lost the real Tier name here.  Is that OK?  (Feels like a gotcha for when this is eventually merged into a multi-tier system.)